packages/module/src/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+// ============================================================================
+// Chatbot Footer
+// ============================================================================
+
+// Global footer component
+// Supports:
+// - Message bar
+// - Footnote
+// - Footnote popover
+
+import React from 'react';
+
+export interface FooterProps {
+  /** Children for the Footer that supports MessageBar and FootNote components*/
+  children?: React.ReactNode;
+}
+
+export const Footer: React.FunctionComponent<FooterProps> = ({ children }) => (
```

Comment:
This should be updated to: 
```
export const Footer: React.FunctionComponent<FooterProps> = ({ 
  children,
  className,
  ...props
}: FooterProps) => (
```

it's the `:FooterProps` in that last line which was missing and preventing the API/props docs from being generated

then the props should be spread to the footer component's div as well

Reply:
Updated. Now the docs are showing all the props. Thanks.